app/src/main/java/ch/epfl/balelecbud/map/MapViewActivity.java

```diff
@@ -1,58 +1,77 @@
 package ch.epfl.balelecbud.map;
 
+import android.graphics.Bitmap;
+import android.graphics.drawable.BitmapDrawable;
+import android.graphics.drawable.Drawable;
 import android.os.Bundle;
 import android.util.Log;
 
 import androidx.annotation.NonNull;
 import androidx.annotation.VisibleForTesting;
+import androidx.core.content.ContextCompat;
 
-import com.google.android.gms.maps.MapView;
+import com.mapbox.mapboxsdk.Mapbox;
+import com.mapbox.mapboxsdk.annotations.Icon;
+import com.mapbox.mapboxsdk.annotations.IconFactory;
+import com.mapbox.mapboxsdk.maps.MapView;
 
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.CompletableFuture;
 
+import ch.epfl.balelecbud.BalelecbudApplication;
 import ch.epfl.balelecbud.BasicActivity;
 import ch.epfl.balelecbud.R;
 import ch.epfl.balelecbud.friendship.FriendshipUtils;
+import ch.epfl.balelecbud.location.LocationUtil;
 import ch.epfl.balelecbud.models.Location;
 import ch.epfl.balelecbud.models.User;
 import ch.epfl.balelecbud.util.database.DatabaseWrapper;
 
 import static ch.epfl.balelecbud.BalelecbudApplication.getAppAuthenticator;
 import static ch.epfl.balelecbud.BalelecbudApplication.getAppDatabaseWrapper;
-import static ch.epfl.balelecbud.location.LocationUtil.isLocationActive;
 
 public class MapViewActivity extends BasicActivity implements OnMapReadyCallback {
```

Comment:
We don't really need to have our own `OnMapReadyCallback` interface since is it never used as a parameter type.